resources/rdf_dataset_exploration_queries/bq6_measure_target_groups.rq

```diff
@@ -0,0 +1,15 @@
+# What are the target groups of the measure?
+
+PREFIX sc: <http://publications.europa.eu/resource/ontology/sc#>
+PREFIX dct: <https://www.dublincore.org/specifications/dublin-core/dcmi-terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+
+SELECT DISTINCT ?measure ?title ?target_l1 ?target_l1_label
+WHERE {
+  ?measure a sc:Measure .
+  ?measure dct:title ?title .
+  ?measure sc:targetGroup ?target_l1 .
+  ?target_l1 skos:prefLabel ?target_l1_label .
+  ?target_l1 skos:inScheme ?scheme_l1 .
+  FILTER(regex(str(?scheme_l1), "PWDBTargetGroupL1" )) .
```

Comment:
remove the regex, and instead of ?schema_l1 provide the URI of that concept scheme.